hms-catalog/pom.xml

```diff
@@ -0,0 +1,264 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why do we create a root-level module? can we move this under
`standalone-metastore/rest-catalog`

Reply:
in Hive for Iceberg tables we support HiveCatalog & HadoopCatalog, for external and ACID tables we have. different interface.

Reply:
We are adding an Iceberg Catalog REST service to Hive Metastore so 3rd party query engines can rely on HMS to access the Iceberg metadata. (https://github.com/apache/iceberg/blob/main/open-api/rest-catalog-open-api.yaml)

Reply:
why can't we expose via REST existing HiveCatalog, why do we need to embed something into HMS?